src/pages/AddPage.tsx

```diff
@@ -5,10 +5,21 @@ import { AddTask } from "../components/model";
 
 const AddPage: React.FC = () => {
   const [addTask, setAddTask] = React.useState<string>("");
-  const [taskTitle, setTaskTitle] = React.useState<AddTask[]>([]);
+  const [addTasks, setAddTasks] = React.useState<AddTask[]>([]);
+  const [taskTitle, setTaskTitle] = React.useState<string>("");
+  const [taskDescription, setTaskDescription] = React.useState<string>("");
+  const [taskFrequency, setTaskFrequency] = React.useState<string>("");
 
   const handleAdd = (e: React.FormEvent) => {
     e.preventDefault();
+    // question: addpage > to add a new kitchen page
```

Comment:
when `handleAdd` is called, `taskTitle` `taskDescription` and `taskFrequency` should have values.  you can use them here to do something like:

```
setAddTask({
title: taskTitle,
description: taskDescription,
frequency: taskFrequency
})
```

Reply:
Tried doing something like this but getting a type error: 

_Argument of type '{ title: string; description: string; frequency: string; }' is not assignable to parameter of type 'SetStateAction<string>'.
  Object literal may only specify known properties, and 'title' does not exist in type '(prevState: string) => string'.ts(2345)_

tried changing the type to an array but then it says it's not finding 'title' should I pass title, description and frequency into the addpage?